app/src/main/java/com/keylesspalace/tusky/components/timeline/viewmodel/NetworkTimelineViewModel.kt

```diff
@@ -211,7 +233,9 @@ class NetworkTimelineViewModel @Inject constructor(
 
     override fun fullReload() {
         statusData.clear()
+        nextKey = null
         currentSource?.invalidate()
+        // FIXME: auto reload needed
```

Comment:
I cannot find an ideal way to request refresh to mediator, so timeline won't loaded after `fullReload` and error message 'Nothing here' will be shown.

Reply:
Try setting the `nextKey` to the id of the first status in `statusData` before clearing it

Reply:
Thanks, the problem seems to be solved